### PR TITLE
upgrade required zend framework modules

### PR DESCRIPTION
**Pull request by Jdstuart13**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+    "keywords": [
+        "zend",
+        "job",
+        "queue"
+    ],
+    "homepage": "http://ivantagehealth.com/",
+    "require": {
+        "php": ">=5.3.3",
+        "zendframework/zend-mvc": "^3.1",
+        "zendframework/zend-db": "^2.11"
```
Comment (evsheffield) [inline composer.json]: Do you know if this package is needed? I don't recall using anything from the Zend\Db namespace.

**Review by evsheffield — CHANGES_REQUESTED** [review]
Thanks for putting this together, James!

I've left one question about Zend\Db that I'd appreciate your thoughts on. Also, as a general cleanup task would you mind reverting all of the files that have white space-only changes so that they aren't included in the final changeset?

File: config/module.config.php [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
-
-	'view_manager' => array(
-		'strategies' => array(
-			'ViewJsonStrategy'
-		)
-	)
+<?php
+
+use Zend\Db\Adapter\Adapter;
+
```
Comment (Jdstuart13) [inline config/module.config.php]: This is where zend/db is used but on a closer look it doesn't seem to be used

Comment (evsheffield) [reply config/module.config.php]: Thanks! Yeah, looks like that isn't used. We can go ahead and remove the `use` and the `zend-db` package requirement.

**Comment by evsheffield** [comment]
Thanks for the updates, James! On last request, let's go ahead and removing that unused `use` statement that you found referencing `Zend\Db`.

**Comment by Jdstuart13** [comment]
I forgot to check that in, whoops

**Review by evsheffield — APPROVED** [review]

**Comment by evsheffield** [comment]
Looks good! Once merged you can create a `v2.0.0` tag.

**Pull request merged**